operator/builtin/output/googlecloud/google_cloud.go

```diff
@@ -52,6 +52,7 @@ type GoogleCloudOutputConfig struct {
 	CredentialsFile     string          `json:"credentials_file,omitempty" yaml:"credentials_file,omitempty"`
 	ProjectID           string          `json:"project_id"                 yaml:"project_id"`
 	LogNameField        *entry.Field    `json:"log_name_field,omitempty"   yaml:"log_name_field,omitempty"`
+	Location            *entry.Field    `json:"location,omitempty"   yaml:"location,omitempty"`
```

Comment:
Optional: Consider calling this `location_field` to match `log_name_field`.

Reply:
I went back and forth on this, I agree. Done.